R/read_target_file.R

```diff
@@ -27,10 +27,13 @@
 #' # read in oracle-output file
 #' read_target_file("oracle-output.csv", hub_path)
 #' read_target_file("oracle-output.csv", hub_path, coerce_types = "chr")
-read_target_file <- function(target_file_path, hub_path,
-                             coerce_types = c("hub", "chr", "none"),
-                             date_col = NULL,
-                             na = c("NA", "")) {
+read_target_file <- function(
+  target_file_path,
+  hub_path,
+  coerce_types = c("target", "chr", "none"),
+  date_col = NULL,
+  na = c("NA", "")
+) {
```

Comment:
The documentation needs to be update for the `coerce_types`, it's still "hub", "chr" or "none" in the documentation.

Reply:
Excellent catch! Fixed in dee3fd3240f365d8257a576628aa87fffbd4d8d9